files/graphdb.service

```diff
@@ -8,7 +8,7 @@ Restart=on-failure
 RestartSec=5s
 User=graphdb
 Group=graphdb
-Environment="GDB_JAVA_OPTS=-Dgraphdb.home=/var/opt/graphdb/node -Dgraphdb.home.conf=/etc/graphdb -Dhttp.socket.keepalive=true -XX:MaxRAMPercentage=85.0 -XX:-UseCompressedOops"
+EnvironmentFile="/etc/environments.d/gdb_environment"
```

Comment:
see [this](https://github.com/Ontotext-AD/terraform-aws-graphdb/pull/21/files#r1389356662) comment and change if needed.